update.c

```diff
@@ -309,37 +396,32 @@ update_locale (XdgDirEntry *old_entries)
         cmdline = XDG_USER_DIRS_UPDATE " --force";
 
       if (!g_spawn_command_line_sync (cmdline, NULL, NULL, &exit_status, NULL) ||
-	  !WIFEXITED(exit_status) ||
-	  WEXITSTATUS(exit_status) != 0)
-	{
-	  GtkWidget *error;
-
-	  error = gtk_message_dialog_new (NULL, 0,
-					  GTK_MESSAGE_ERROR,
-					  GTK_BUTTONS_OK,
-					  _("There was an error updating the folders"));
-	  
-	  gtk_dialog_run (GTK_DIALOG (error));
-	  gtk_widget_destroy (error);
-	}
+          !WIFEXITED(exit_status) ||
+          WEXITSTATUS(exit_status) != 0)
+        {
+          show_error_dialog (_("There was an error updating the folder"));
```

Comment:
You're still changing the translatable string (`folders` -> `folder`). I think there should be no need to change this code at all here, since we're not adding another error message.